src/redux/configureStore.js

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable */
+
```

Comment:
- [ ] For best practice, kindly make sure you fix your linters errors instead of dasabling them for the enire file.